src/views/texts/TextDetails.vue

```diff
@@ -111,6 +115,23 @@ export default class TextDetails extends Vue {
 				this.snackbarText = 'Error while adding text to the project';
 			});
 	}
+
+	private async removeText(): Promise<void> {
+		Vue.$axios.delete(`/project/${this.$route.query.project_id}/delete_text`, {
+				data: { text_id: this.masterId },
+			})
+			.then((response: AxiosResponse) => {
+				this.getTextDetails();
+			})
+			.catch((error) => {
+				this.snackbar = true;
+				if (error.response.status === 412) {
```

Comment:
You can leave this but it should first check if a text has annotations that have been submitted to quadriga and if it does it should not show the button to begin with. The error message is a good fallback in case something weird happens.

Reply:
This has not been resolved. The button still appears even if a text has an appellation that has been submitted.